src/shims/unix/linux/fd/event.rs

```diff
@@ -59,11 +61,17 @@ impl FileDescriptor for Event {
         bytes: &[u8],
     ) -> InterpResult<'tcx, io::Result<usize>> {
         let v1 = self.val.get();
-        // FIXME handle blocking when addition results in exceeding the max u64 value
-        // or fail with EAGAIN if the file descriptor is nonblocking.
-        let v2 = v1.checked_add(u64::from_be_bytes(bytes.try_into().unwrap())).unwrap();
+        let bytes = bytes
+            .try_into()
+            .map_err(|_| err_unsup_format!("we expected 8 bytes and got {}", bytes.len()))?;
+        let v2 = v1.checked_add(u64::from_be_bytes(bytes)).ok_or_else(|| {
+            err_unsup_format!(
+                "handle blocking when addition results \
+                in exceeding the max u64 value or fail with \
+                EAGAIN if the file descriptor is nonblocking."
```

Comment:
Should this say something more like "overflow not implemented", if it's left this way? That way the message isn't aspiration & reflects the current state. I could put the `FIXME` comment back so it's known what needs to be implemented still.

Reply:
The error message should be useful to a user seeing this come out of Miri for the first time, and I think it should be worded as if you're saying something to the user.

Currently this is worded more like a note to you/us. I'd go for something like 

> Miri currently has an incomplete epoll implementation. This operation would overflow (please fill in what exactly is being overflowed here, the details of this are a bit beyond me at the moment), which is not yet implemented.